eva-ingest-remapped-variants/src/test/resources/ingest-remapped-variants.properties

```diff
@@ -0,0 +1,26 @@
+parameters.vcf=src/test/resources/input-files/vcf/vcf_out_with_header.vcf
+parameters.assemblyAccession=GCA_000000001.2
+parameters.remappedFrom=GCA_000000001.1
+parameters.chunkSize=2
+
+accessioning.instanceId=instance-01
+accessioning.submitted.categoryId=ss
+accessioning.clustered.categoryId=rs
+
+accessioning.monotonic.ss.blockSize=100000
+accessioning.monotonic.ss.blockStartValue=5000000000
+accessioning.monotonic.ss.nextBlockInterval=1000000000
+accessioning.monotonic.rs.blockSize=100000
+accessioning.monotonic.rs.blockStartValue=3000000000
+accessioning.monotonic.rs.nextBlockInterval=1000000000
```

Comment:
Although unncessary, the "accessioning.*" properties might be a necessary evil to satisy Spring autowiring gods but @andresfsilva will take another look to see if these can be dispensed with.

Reply:
I think this should be done but will require a new configuration. Added in ticket [EVA-420](https://www.ebi.ac.uk/panda/jira/browse/EVA-2420)